services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:

```

Comment:
If argument can be None, you should show it in type hint. Fix everywhere
```suggestion
def create_order(tickets: list[dict], username: str, date: str | None= None) -> None:

```